posts/2019-05-23-Rust-1.35.0.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: "Announcing Rust 1.35.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.35.0. Rust is a
+programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust 1.35.0 is as easy as:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the appropriate page on our website,
+and check out the [detailed release notes for 1.35.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1350-2019-05-23
+
+## What's in 1.35.0 stable
+
+The highlight of this release is the implementation of the `FnOnce`, `FnMut`,
+and `Fn` closure traits for `Box<dyn FnOnce>`, `Box<dyn FnMut>`, and `Box<dyn Fn>` respectively.
```

Comment:
Is it worth clarifying at some point that it's for `Box<F>` where `F: Fn* + ?Sized`?

Reply:
@Manishearth ^---

Reply:
Oh, yeah, can someone do that?